Makefile.x86_64

```diff
@@ -8,6 +8,20 @@ endif
 endif
 endif
 
+
+ifndef DYNAMIC_ARCH
+define ADD_CPUFLAGS
+1
+endef
```

Comment:
```suggestion
endif
```

Reply:
nope, that's the end statement closing the define (but using a normal assignment is probably clearer)